Includes/Runeterra/Helpers/DeckCodeHelpers.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Runeterra Team
+// Chris Ohk, Seokmin Hong
+// We are making my contributions/submissions to this project solely in our
+// personal capacity and are not conveying any rights to any intellectual
+// property of any third parties.
+
+#ifndef RUNETERRA_DECK_CODE_HELPERS_HPP
+#define RUNETERRA_DECK_CODE_HELPERS_HPP
+
+#include <string>
+#include <vector>
+
+namespace Runeterra::DeckCode
+{
+//! Decodes a deck code and returns the deck that contains some cards.
+//! \param deckCode The deck code generated by Legends of Runeterra.
+//! \return A list of cards that is decoded.
+std::vector<std::string> Decode(const std::string& deckCode);
+
+//! Decodes a group and returns a list of matched cards.
+//! \param cardStream A card stream for decoded base32 string.
+//! \param deck A deck instance to add card(s).
+//! \param amount The number of card(s) to add.
+void DecodeGroup(std::vector<uint8_t>* cardStream,
```

Comment:
How about moving the definition of the `DecodeGroup` function to the `DeckCodeHelpers.cpp` file? Because it is assumed to be used in the body of the `Decode` function.

Reply:
Alright. I agree, too. I'll change it. 😄